pkg/resources/user_acceptance_test.go

```diff
@@ -76,6 +76,11 @@ func TestAcc_User(t *testing.T) {
 			// RENAME
 			{
 				Config: uConfig(prefix2, sshkey1, sshkey2),
```

Comment:
rename + some other change to validate correctness?

Reply:
Hmm, here also login_name changes, but I added a comment change for good measure.